server/services/database/documents.js

```diff
@@ -108,11 +108,23 @@ const checkDocumentId = documentId => {
     .catch(error => console.error(error));
 };
 
+const getDocumentIdByTranslationId = translationId => {
+  const query = "select document_id from translations where id =$1";
+  return pool
+    .query(query, [translationId])
+    .then(result => result.rows)
+    .catch(e => {
+      console.log("this is new error !!!");
+      console.error(e);
```

Comment:
You should not catch the error to console.log it, it will silently fail otherwise. Catch the error in your route implementation and return an error and a 400 status code there.